src/aliceVision/sfmData/SfMData.cpp

```diff
@@ -18,6 +20,14 @@ using namespace aliceVision::image;
 
 namespace fs = boost::filesystem;
 
+SfMData::SfMData()
+{
+}
```

Comment:
```suggestion
SfMData::SfMData() = default;
```

Reply:
and move it to the hpp

Reply:
I made an explicit constructor in the cpp has it creates a link error if the dependencies are not correct (as before).
And this also allows to put a break point in debug to trace allocation/desallocation of the SfMData. So I decided to commit it at the end as it may be useful in multiple situations.